docs/development/workflow/get_devel_version.rst

```diff
@@ -229,6 +229,16 @@ ahead to the more sophisticated method look at :ref:`virtual_envs`).
     + You use Python 3.
     + You want to work on C or Cython code in `Astropy`_.
 
+Now is the time to check that you have all the required dependencies installed on 
+your machine to make sure the pip install would be a successfully one. Namely, as 
+you are effectively building from source, please check that you have both "cython"
+as well as "jinja2" installed using pip. (Note: Using conda to installed these two 
+dependencies would normally not work.) Please note that by pip we really mean pip3. 
+If you have any confusing regarding what is the default pip version you have, try 
```

Comment:
I think you might be best off just writing something like "... jinja2 installed. It may be easiest to use ``pip``. Here, we mean the python3 version of ``pip``; on your machine, this may be called ``pip3``.

I would probably not give the `alias` line - people either know how to do this or can replace `pip` with `pip3` themselves (and `alias` works slightly differently under different shells, and not like this on windows).

Reply:
Yup, agreed. The explicit `alias` reference has been removed, and `pip3` wording has been changed to "python version of `pip`" in the passage.